include/uxr/agent/transport/tcp/TCPServerLinux.hpp

```diff
@@ -44,7 +44,7 @@ class TCPServer : public TCPServerBase
     ~TCPServer() = default;
 
 private:
-    virtual bool init() override;
+    virtual bool init(bool discovery_enabled) override;
```

Comment:
To keep consistency remove `virtual` from derived class overridden methods. If the derived class should not have more derived, consider `final` keyword.

Inconsistency between SerialServers and TCP and UDP servers.

Reply:
What is exactly the inconsistency?